conf/puppet/hieradata/common.yaml

```diff
@@ -17,6 +17,7 @@ aem_curator::install_author::setup_repository_volume: true
 aem_curator::install_author::repository_volume_mount_point: /mnt/ebs1
 aem_curator::install_author::post_stop_sleep_secs: 300
 aem_curator::install_author::aem_license_base: "%{lookup('tmp_dir')}/license"
+aem_curator::install_author::aem_keystore_password: "%{lookup('AEM_Keystore_Password')}"
```

Comment:
Is `AEM_Keystore_Password` name controlled by Packer AEM during create aws resources step?

Reply:
No, it is a static value, I assume we want Ansible to continue to create the parameter store resource and add an additional value to the jinja2 template that passes whatever the parameter name is to puppet's hiera.

Let me know if this is acceptable.